### DISCUSSION TITLE
Visual Studio 2022 setup

### DISCUSSION THREAD

**Discussion by a-electronix** [cesanta/mongoose#2909] [Ask questions, get answers] [answered]
Hi,
I am trying to test mongoose webserver together with an existing a visual studio project 2022 on a windows 11 machine.
I have used one of the example projects for doing some test outside the visual studio setup and that is working fine and I guess there are some of my VS project setup that is incorrect.
I have a lot of errors in the winsock2.h header file.
Br. Martin

**Comment by scaprile** [comment, marked as answer]
"a lot of errors" is not very precise nor helpful.
https://github.com/cesanta/mongoose/blob/9b0d11217605cb5c6446e261d6b5940bed0a5d31/tutorials/http/http-server/Makefile#L13-L19
https://mongoose.ws/documentation/tutorials/tools/
https://github.com/cesanta/mongoose/blob/9b0d11217605cb5c6446e261d6b5940bed0a5d31/test/Makefile#L13
https://github.com/cesanta/mongoose/blob/9b0d11217605cb5c6446e261d6b5940bed0a5d31/test/Makefile#L177-L178
If there is an actual issue, please open an issue, _honor the issue template_ so we can know your scenario and reproduce the issue if necessary, and let us know _exactly_ what you are doing
